### DISCUSSION TITLE
Can't edit after upgrade.

### DISCUSSION THREAD

**Discussion by scorrell11** [outline/outline#8000] [Self Hosting] [answered]
I am upgrading from v0.67.2 and I lose the ability to edit pages. I can edit the title but that is it. I have tried upgrading to each version and v0.68.0 and v0.68.1 work fine but as soon as I upgrade to v0.69.0 the ability to edit does not work. I get the same results if I try to upgrade directly to the latest version as well. I have enabled and disabled Seamless editing but the issue remains.

**To Reproduce**
1. Update to v0.69.0
2. Click edit on any collection
3. click on page and nothing happens

**Outline (please complete the following information):**
- Install: self hosted]
- Version: 0.69.0

**Desktop (please complete the following information):**
 - OS: Windows / Linux
 - Browser: Chrome, Edge & Firefox

**Comment by scorrell11** [comment, marked as answer]
COLLABORATION_URL was unset which by default uses ws:// which works up through v0.68.1. v0.69.0 and up require wss:// for the COLLABORATION_URL. Setting this to wss:// allowed the edits again.

**Reply by tommoor** [reply]
Interesting! given this version is over a year old there isn't anything to change at this point

**Comment by tx1ee** [comment]
I wonder if this problem has been solved? I also encountered the same problem with the latest docker version.

**Reply by jorgegarciadev** [reply]
No, it's no solved. I can only edit documents in Chrome or Chrome based browsers like Brave. Firefox, Safari or Orion don't work.

**Comment by axlroden** [comment]
sorry to necro, but this showed up on search.

COLLABORATION_URL was not set on my instance, and I had issues in the Brave browser, but no other browser(chrome, firefox, opera), where i couldn't click on text to edit, only the title.
After i set env var COLLABORATION_URL="" then it started working again .. 
repository: docker.io/outlinewiki/outline
tag: 0.83.0

**Reply by tommoor** [reply]
Very strange, unset and empty string are treated exactly the same

**Reply by axlroden** [reply]
could also just have been a restart of instance then
